### PR TITLE
Add ctor, method and property tests to UTF7Encoding

### PR DESCRIPTION
**Pull request by hughbe**
- Ctor, Ctor_Bool
- EncoderFallback, DecoderFallback
- WebName, CodePage, EncodingName, IsSingleByte
- Clone, Equals, GetPreamble

Also removes some files that VS didn't delete in previous cleanups

/cc @tarekgh

### CODE REVIEW

File: src/System.Text.Encoding/tests/UTF7Encoding/UTF7EncodingTests.cs [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+            UTF7Encoding encoding = new UTF7Encoding(allowOptionals);
+            VerifyUtf7Encoding(encoding, allowOptionals);
+        }
+
+        public static void VerifyUtf7Encoding(UTF7Encoding encoding, bool allowOptionals)
+        {
+            Assert.Empty(encoding.GetPreamble());
+
+            Assert.Equal(new EncoderReplacementFallback(string.Empty), encoding.EncoderFallback);
+            DecoderFallback decodeFallback = encoding.DecoderFallback;
```
Comment (tarekgh) [inline src/System.Text.Encoding/tests/UTF7Encoding/UTF7EncodingTests.cs]: it looks this is not used.

Comment (hughbe) [inline src/System.Text.Encoding/tests/UTF7Encoding/UTF7EncodingTests.cs]: god catch, I got rid of it

**Comment by tarekgh** [comment]
LGTM. thanks.

**Comment by tarekgh** [comment]
test Innerloop Ubuntu14.04 Release Build and Test please

**Pull request merged**